Xamarin.Forms.Platform.Android/AppCompat/MasterDetailContainer.cs

```diff
@@ -106,6 +107,37 @@ protected override void AddChildView(VisualElement childView)
 			}
 		}
 
+		protected override void Dispose(bool disposing)
+		{
+			if (_disposed)
+			{
+				return;
+			}
+
+			_disposed = true;
```

Comment:
Can you make your dispose pattern the same as others? `_disposed` should be set inside the `disposing` block in case `Dispose()` is called with a false flag.